topostats/entry_point.py

```diff
@@ -373,6 +373,13 @@ def create_parser() -> arg.ArgumentParser:
         required=False,
         help="Extract height profiles along maximum feret of molecules",
     )
+    process_parser.add_argument(
+        "--grainstats-class-names",
+        dest="grainstats_class_names",
+        type=list,
```

Comment:
I forgot to look this up the other day as I've never done it but `add_argument()` needs more than just the `type=list` option it needs the `nargs=` option, probably with the value `"+"` to indicate that there has to be one or more argument (i.e. `nargs="+"`) and these are then stored in a `list`.

Reply:
Sorry for not being explicit, the same option needs adding to the `grainstats_parser()` (but see comment below at that point).